### ISSUE TITLE
make the ews root dir and theme configurable options for those using spacemacs etc.

### ISSUE THREAD

**Issue by vxe** [pprevos/emacs-writing-studio#29]
great work on this, I was able to load this from my spacemacs configs by manually overriding the ews.el location, which is assumed to be .emacs.d and commenting on the theme block.  If you could add a defcustom which default to ~/.emacs.d but is configurable to a custom path I think you could support people coming from other distributions without breaking any current users.  thanks

**Comment by pprevos** [comment]
Next week I will publish `ews.el` as a separate package, so it can be pulled from MELPA. Would that help your case?

**Comment by vxe** [comment]
yeah that would be great! and if possible making loading the provided theme optional as it collided with my current one that would also be appreciated. thanks

**Comment by pprevos** [comment]
I never intended the EWS configuration to be loaded as is. It is a starter kit that you can modify to your liking. I will only update it sporadically.

**Comment by zenny** [comment]
> great work on this, I was able to load this from my spacemacs configs by manually overriding the ews.el location, which is assumed to be .emacs.d and commenting on the theme block. If you could add a defcustom which default to ~/.emacs.d but is configurable to a custom path I think you could support people coming from other distributions without breaking any current users. thanks

Hi @vxe ,

From emacs v29, it is possible to specify init directory to the desired location. I use:

`emacs --debug-init --init-directory=/path/to/emacs-writing-studio` without messing the default `~/.emacs.d` directory. It works like a charm.

Happy new year and cheers,
/zenny
